### ISSUE TITLE
pre-trained model of Swin + VSA

### ISSUE THREAD

**Issue by lyu124** [ViTAE-Transformer/ViTAE-VSA#6]
hello, i am now doing a work about semantic segmentation and i would like to use swin with vsa model as the feature extractor backbone. I would like to ask,  about when will you maybe upload the pre-trained model? Without pretrained model the results are really with low accuracy now... Thank you so much.
